### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by joykare**
Added circleCI

### CODE REVIEW

File: jasmine/spec/string_manipulation_test.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ describe('toUpper', function(){
     expect('lol'.toUpper()).toBe('LOL');
     expect('YEAY'.toUpper()).toBe('YEAY');
     expect('nOP'.toUpper()).toBe('NOP');
+    expect('123roy'.toUpper()).toBe('123ROY');
```
Comment (joykare) [inline jasmine/spec/string_manipulation_test.js:17]: ## Code Quality - Nitpick

Add test to check whether numbers affect the toUpper() method

File: jasmine/spec/string_manipulation_test.js [hunk 2]
```diff
@@ -68,6 +69,7 @@ describe('toCurrency', function(){
     expect('1000'.toCurrency()).toBe('1,000.00');
     expect('1000.50'.toCurrency()).toBe('1,000.50');
     expect('1000000'.toCurrency()).toBe('1,000,000.00');
+    expect('123'.toCurrency()).toBe('123.00');
```
Comment (joykare) [inline jasmine/spec/string_manipulation_test.js:72]: ## Code Quality - Nitpicks

Add test for a string that will not need commas

File: src/string_manipulation.js [hunk 3]
```diff
@@ -30,14 +30,13 @@
 	};
 
 	String.prototype.wordCount = function() {
-		var stringArray = this.words();
-		return stringArray.length;
+		return this.words().length;
```
Comment (joykare) [inline src/string_manipulation.js:33]: ## Code Quality - Nitpicks

Could be a one liner

File: jasmine/spec/string_manipulation_test.js [hunk 4]
```diff
@@ -68,6 +69,7 @@ describe('toCurrency', function(){
     expect('1000'.toCurrency()).toBe('1,000.00');
     expect('1000.50'.toCurrency()).toBe('1,000.50');
     expect('1000000'.toCurrency()).toBe('1,000,000.00');
```
Comment (johnkariuki) [inline jasmine/spec/string_manipulation_test.js:71]: What about a test for a figure that will not have commas added? Say `123`

File: src/string_manipulation.js [hunk 5]
```diff
@@ -30,14 +30,13 @@
 	};
 
 	String.prototype.wordCount = function() {
-		var stringArray = this.words();
```
Comment (johnkariuki) [inline src/string_manipulation.js:33]: Make this a one liner? Looks much neater.

**Pull request merged**